pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/BrokerServerBuilder.java

```diff
@@ -57,23 +57,14 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-
-/**
- * Aug 4, 2014
- */
 public class BrokerServerBuilder {
   private static final String TRANSPORT_CONFIG_PREFIX = "pinot.broker.transport";
   private static final String CLIENT_CONFIG_PREFIX = "pinot.broker.client";
   private static final String METRICS_CONFIG_PREFIX = "pinot.broker.metrics";
-  private static final String BROKER_TIME_OUT_CONFIG = "pinot.broker.timeoutMs";
+  private static final long DEFAULT_BROKER_DELAY_SHUTDOWN_TIME_MS = 10 * 1000L;
```

Comment:
This just moved to BrokerRequestHandler.java

This class was reading from config and passing it to BrokerRequestHandler. The constructor had too many such parameters. I replaced this with config object and changed BrokerRequestHandler to read required info from config.